test/README.md

```diff
@@ -66,6 +66,11 @@ mode, use
 ```bash
 test/local-e2e-tests.sh -short
 ```
+### e2e prow jobs
```

Comment:
`E2E Prow jobs` looks better? I can't decide. :)

Reply:
changed and updated all headings to match the style. :)